client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCart.tsx

```diff
@@ -1,6 +1,7 @@
 import React, {ReactElement, useState, useEffect} from 'react';
 import {useLocation, useNavigate} from 'react-router';
 import {EventRow} from './AdminPurchase';
+import {dayMonthDate, militaryToCivilian} from '../../../../utils/arrays';
```

Comment:
Does this import need to be added? Since you are using it in the code you changed?

Reply:
nope! removed.